pkg/reconciler/broker/broker_test.go

```diff
@@ -58,10 +58,7 @@ const (
 	triggerChannel channelType = "TriggerChannel"
 	ingressChannel channelType = "IngressChannel"
 
-	triggerChannelName = "test-broker-kn-trigger"
-	ingressChannelName = "test-broker-kn-ingress"
-
-	ingressCRDChannelName = "test-broker-kn2-ingress"
+	ingressCRDChannelName = "test-broker-kne-ingress"
```

Comment:
```suggestion
	ingressChannelName = "test-broker-kne-ingress"
```

As we no longer need the distinction between CRD and non-CRD Channels.